mysql/src/test/scala/org/scalarelational/mysql/MysqlTestDataStore.scala

```diff
@@ -0,0 +1,33 @@
+package org.scalarelational.mysql
+
+import org.scalarelational.column.property._
+import org.scalarelational.model.{SQLLogging, Table}
+import org.scalarelational.column.property.ForeignKey
+import org.powerscala.log.Level
+
+case class Supplier(name: String, street: String, city: String, state: String, zip: String, id: Option[Int] = None)
+case class Coffee(name: String, supID : Option[Int], price: Double, sales: Int, total : Int , id: Option[Int])
+
+object MysqlTestDataStore extends MysqlDatastore("jdbc:mysql://localhost/test", "root", "m") with SQLLogging{
```

Comment:
Yeah, it would be nice if it had some additional features to easily generate the URL. Perhaps taking a case class:

```scala
case class MySQLConfig(host: String, schema: String)
```

and perhaps any other optional settings. I do something similar in H2 but H2 has a crazy number of properties I can set.